backend/apps/question/src/question.service.ts

```diff
@@ -67,7 +69,10 @@ export class QuestionService {
       })) ?? (await this.diffcultyModel.findById(difficulty.toString()));
 
     if (difficultyObject == null) {
-      throw new NotFoundException('Difficulty not found');
+      throw new PeerprepException(
+        'Difficulty not found',
+        PEERPREP_EXCEPTION_TYPES.NOT_FOUND,
```

Comment:
Is it possible to be more specific on the type of NOT_FOUND exception, for instance NOT_FOUND_DIFFICULTY, NOT_FOUND_CATEGORY, etc.